src/Microsoft.AspNet.Mvc.Localization/LocalizedHtmlString.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Rendering;
+
+namespace Microsoft.AspNet.Mvc.Localization
+{
+    /// <summary>
+    /// A <see cref="HtmlString"/> with localized content.
+    /// </summary>
+    public class LocalizedHtmlString : HtmlString
```

Comment:
Mildly related to this PR - should we have an `IHtmlString` instead? We don't actually care about the behavior of `HtmlString` here - just that Razor treats types it specially which is better achieved via marker interfaces